packages/editor-ui/src/components/ContactPromptModal.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+	<Modal
+		:name="modalName"
+		:eventBus="modalBus"
+		:center="true"
+		:closeOnPressEscape="false"
+		:customClass="'contact-prompt-modal'"
+		:beforeClose="closeDialog"
+		title="You're a power user"
+		minWidth="460px"
+		maxWidth="460px"
+	>
+		<template slot="header">
+			<h2 :class="$style.title">You’re a power user 💪</h2>
```

Comment:
can you try to use heading component https://n8n-io.github.io/n8n-storybook/?path=/story/atoms-heading--heading

Reply:
@mutdmour I would love to use heading and text components. But I cannot pass any class to the component. Any suggestion on this? :)

Reply:
Done after your directions from yesterday :)